docs/content/Coding Milestones/PoC2023q1/example1-subs/example1-post-kcp.md

```diff
@@ -82,7 +82,7 @@ containerized workloads from the four resources built into kcp TMC
 are meaningful in KubeStellar.
 
 ```shell
-kubestellar init
+IN_CLUSTER=false SPACE_MANAGER_KUBECONFIG=$SM_CONFIG kubestellar init
```

Comment:
Is `IN_CLUSTER=false` the right choice here? If there is a difference between the two kubeconfigs, wouldn't the inner-more one be more aligned with what is going on here?